cristin-organization/src/main/java/no/unit/nva/cristin/organization/FetchCristinOrganizationHandler.java

```diff
@@ -0,0 +1,118 @@
+package no.unit.nva.cristin.organization;
+
+
+import com.amazonaws.services.lambda.runtime.Context;
+import no.unit.nva.cristin.model.Organization;
+import no.unit.nva.utils.Language;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.apigateway.exceptions.BadRequestException;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.paths.UriWrapper;
+
+import java.net.HttpURLConnection;
+import java.net.URI;
+import java.util.Optional;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+import static no.unit.nva.cristin.projects.Constants.BASE_PATH;
+import static no.unit.nva.cristin.projects.Constants.DOMAIN_NAME;
+import static no.unit.nva.cristin.projects.Constants.HTTPS;
+import static no.unit.nva.cristin.projects.Constants.IDENTIFIER;
+import static no.unit.nva.cristin.projects.Constants.LANGUAGE;
+import static no.unit.nva.cristin.projects.Constants.ORGANIZATION_PATH;
+import static no.unit.nva.cristin.projects.ErrorMessages.ERROR_MESSAGE_INVALID_PATH_PARAMETER_FOR_ID;
+import static no.unit.nva.cristin.projects.ErrorMessages.ERROR_MESSAGE_INVALID_QUERY_PARAMS_ON_LOOKUP;
+import static no.unit.nva.cristin.projects.ErrorMessages.ERROR_MESSAGE_LANGUAGE_INVALID;
+import static nva.commons.core.attempt.Try.attempt;
+
+public class FetchCristinOrganizationHandler extends ApiGatewayHandler<Void, Organization> {
+
+    public static final String IDENTIFIER_PATTERN = "^(?:[0-9]{1,3}\\.){3}[0-9]{1,3}$";
+    public static final Pattern PATTERN = Pattern.compile(IDENTIFIER_PATTERN);
+    protected static final String DEFAULT_LANGUAGE_CODE = "nb";
```

Comment:
We should always return every language… (comma-separated list in the HTTP lang-param sent to Cristin API), so language logic isn't needed

Reply:
https://api.cristin.no/v2/projects/435543?lang=en,nb,nn